Extensions/Lighting/lightruntimeobject-pixi-renderer.ts

```diff
@@ -430,6 +454,10 @@ namespace gdjs {
           }
         }
       }
+      // obstaclePolygons[0] === this._lightBoundingPoly
+      // this._lightBoundingPoly === this._object.hitBoxes
+      // This is actually modifying the object hit boxes.
+      // Is it intended?
```

Comment:
I don't think this is intended.

The idea in the algorithm is to always have a rectangle which is at least as large as the light (i.e the rectangle which has a width/height of `2*radius` with as center the center of the light), but if obstacles are outside this rectangle we extend it.

But this should not modify the hitbox. So the renderer here should keep its own `this._lightBoundingPoly` without pointing to the vertices of the object hitbox - which is wrong (but thankfully not creating issues, so far at least)